ibm/service/enterprise/resource_ibm_enterprise_account.go

```diff
@@ -72,6 +74,15 @@ func ResourceIBMEnterpriseAccount() *schema.Resource {
 					},
 				},
 			},
+			"options": {
```

Comment:
Tomorrow if we add some new option support with different type(may be sting/int) then you can't use this argument for feature support
Why we didn't go with list (schema)?